packages/react-native/scripts/react_native_pods.rb

```diff
@@ -248,6 +248,8 @@ def react_native_post_install(
   ReactNativePodsUtils.apply_flags_for_fabric(installer, fabric_enabled: fabric_enabled)
   ReactNativePodsUtils.apply_xcode_15_patch(installer)
   ReactNativePodsUtils.updateIphoneOSDeploymentTarget(installer)
+  ReactNativePodsUtils.updateOSDeploymentTarget(installer)
```

Comment:
`updateOSDeploymentTarget` doesn't exist in `packages/react-native/scripts/cocoapods/utils.rb` on `0.72`.

Reply:
I'm fixing this, thanks for reporting!